src/plugins/intel_gpu/src/kernel_selector/core/common/jitter.cpp

```diff
@@ -1851,6 +1851,27 @@ std::string FusedOpsCodeGenerator::GetJitLoad(const FusedOpsConfiguration& conf,
 
     bool safe_load = conf.boundary_check == FusedOpsConfiguration::BoundaryCheck::ENABLED;
```

Comment:
For commit title: I think it is not about `linear input` but about `planar input`

Reply:
Fixed